src/Commands/Plugin/ShowCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Plugin;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+/**
+ * Manage Terminus plugins.
+ *
+ * @package Pantheon\Terminus\Commands\Plugin
+ */
+class ShowCommand extends PluginBaseCommand
+{
+    /**
+     * List all installed Terminus plugins.
+     *
+     * @command plugin:show
+     * @aliases plugin:list plugin:display
```

Comment:
- List commands are never aliased
- `x:list` commands get an alias of `xs` where possible
- Show and display are not words we've been using in the Terminus commands. You would use `info`. Since this returns a `RowsOfFields` it should be `x:list`/`xs` only.

Reply:
@TeslaDethray: Thank you for the feedback.  It's been so long since I worked on this, I need to spend some time to rethink the process.  I'm sure Terminus has changed from 2 years ago.  So what you are suggesting is this?
```
* @command plugin:list
* @aliases plugin:lists
```